src/ResourceManager/Resources/Commands.Resources/RoleAssignments/ValidateScope.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.WindowsAzure.Commands.Common
+{
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field)]
+    public class ValidateScope : ValidateArgumentsAttribute
+    {
+        protected override void Validate(object arguments, EngineIntrinsics engineIntrinsics)
+        {
+            if (arguments != null)
```

Comment:
As you note, you must add sometest cases, both a scenario test case and some unit tests of this validator